components/prism-dot.js

```diff
@@ -0,0 +1,77 @@
+// https://www.graphviz.org/doc/info/lang.html
+
+(function (Prism) {
+
+	var ID = '(?:' + [
+		// an identifier
+		/\b[a-zA-Z_][\w\x80-\x9F\xA1-\xFF]*/.source,
+		// a number
+		/-?(?:\.\d+|\b\d+(?:\.\d*)?)/.source,
```

Comment:
I think, `ÿ123` (`ÿ` is \xFF) should be interpreted as a single ID.

Reply:
Fixed.